src/workerd/jsg/BUILD.bazel

```diff
@@ -11,6 +11,7 @@ wd_cc_library(
     srcs = [
         "async-context.c++",
         "buffersource.c++",
+        "compile-cache.c++",
```

Comment:
Is it a good opportunity to create to a separate compilation target maybe?

Reply:
I couldn't find a good way to seperate to a different target since compile cache depends on `jsg` and `jsg` depends on compile cache.